content/docs/run-api-server/ingestion.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: Ingestion
+order: 45
+---
+
+Horizon provides access to both current and historical state on the Stellar network through a process called **ingestion**.
+
+Horizon provides most of its utility through ingested data, and your Horizon server can be configured to listen for and ingest transaction results from the Stellar network. Ingestion enables API access to both current (e.g. someone's balance) and historical state (e.g. someone's transaction history).
+
+
+## Ingestion Types
+
+There are two primary ingestion use-cases for Horizon operations:
+
+  - ingesting **live** data to stay up to date with the latest, real-time changes to the Stellar network, and
+  - ingesting **historical** data to peek how the Stellar ledger has changed over time
+
+### Ingesting Live Data
+
+Though this option is disabled by default, in this guide we've [assumed](./configuring.mdx) you turned it on. If you haven't, pass the `--ingest` flag or set `INGEST=true` in your environment.
+
+For a serious setup, **we highly recommend having more than one live ingesting instance**, as this makes it easier to avoid downtime during upgrades and adds resilience to your infrastructure, ensuring you always have the latest network data.
+
+### Ingesting Historical Data
+
+Providing API access to historical data is facilitated by a Horizon subcommand:
+
+<CodeExample>
+
+```
+stellar-horizon db reingest range <start> <end>
+```
+
+</CodeExample>
+
+_(The command name is a bit of a misnomer: you can use `reingest` both to ingest new ledger data and reingest old data.)_
+
+You can run this process in the background while your Horizon server is up. It will continuously decrement the `history.elder_ledger` in your `/metrics` endpoint until the `<end>` ledger is reached and the backfill is complete. If Horizon receives a request for a ledger it hasn't ingested, it returns a 503 error and clarify that it's `Still Ingesting` (see [below](#some-endpoints-are-not-available-during-state-ingestion)).
+
+#### Deciding on how much history to ingest
+(TODO: Think of a better section name; pls suggest something.)
```

Comment:
How about "Strategies for Managing Ingested Data"?

Reply:
Or "Strategies for Managing Historical Data"

Reply:
Actually, you know what: I like yours better: "Deciding how much history to ingest."  I think that's a good title.  It's very clear.

Reply:
Fair enough!